redaxo/src/core/lib/util/editor.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+class rex_editor
+{
+    use rex_factory_trait;
+
+    // see https://github.com/filp/whoops/blob/master/docs/Open%20Files%20In%20An%20Editor.md
+    // keep this list in sync with $supportedEditors in system.settings.php
```

Comment:
Der Kommentar stimmt so nicht mehr, man muss sie hingegen synchron halten mit der Liste weiter unten in dieser Klasse.

Reply:
Done